app/controllers/login_controller.rb

```diff
@@ -1,5 +1,18 @@
+# frozen_string_literal: true
+
 class LoginController < ApplicationController
-    def new
-        
+  def create
+    email_address = params['email']
+    user = User.new
+    if user.user?(email_address)
```

Comment:
Totally different point: it looks like in your current setup you basically ran `user = User.new` just so that you can then access the method on the user instance: `user.user?`. 

In cases like this, a class method might make more sense.

E.g.

```
class User < ApplicationRecord
  validates :name,  :username, :email, presence: true
  validates :email, uniqueness: true

  def user?(email_address)
    User.find_by email: email_address
  end
end

# user = User.new
# user.user?(email)

```

Becomes 
```
class User < ApplicationRecord
  validates :name,  :username, :email, presence: true
  validates :email, uniqueness: true

  def self.user?(email_address)
    User.find_by email: email_address
  end
end

# used like:
# User.user?(email_address)
```

That way you don't need to instantiate an object via new to access the method

Reply:
Thanks for highlighting how to use 'self' - I did not know how to call class methods without instatiating an object

I changed ```user?``` to ```get_user```